eldritch/events.py

```diff
@@ -2361,6 +2354,55 @@ def animated(self):
     return True
 
 
+class LoseSpecific(DiscardSpecific):
```

Comment:
I might do these the other way around, where DiscardSpecific is a subclass of LoseSpecific. This way, things that look for isinstance(event, DiscardSpecific) will not pick up on LoseSpecific, which looks like it is being used to discard blessings/curses without triggering any other events.

Reply:
As discussed, there is no difference between Losing and Discarding, so I'm getting rid of the class.